include/taskolib/Sequence.h

```diff
@@ -239,6 +254,15 @@ class Sequence
      */
     ConstIterator erase(ConstIterator first, ConstIterator last);
 
+    /**
+     * It is guaranteed that each step in the sequence can have a setup script that is
+     * executed before the step executes its script. However it is only relevant of the
```

Comment:
I would reformulate this in the same way as above (no "each step").

Also, the parameter is a script. Maybe we should call it like that and add a description?